Header header

# Gyro reported angle in degrees 0 to 360 degrees
int16 angle
# Gyro reported rate in degrees per second
int16 rate
